lib/StatusIndicator/index.js

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const StatusIndicator = props =>
-  (<span className="status-indicator">
-    <span
-      className="status-indicator__circle"
-      style={{ backgroundColor: props.color }}
-    />
-    <span className="status-indicator__label">{ props.label }</span>
-    <span className="status-indicator__children">{ props.children }</span>
-  </span>);
+const StatusIndicator = (props) => {
+  const classNames = ['status-indicator'];
```

Comment:
Whenever className conditional log gets a little much I recommend we use the `classnames` package which is already in this library.

It turns that conditional logic into the following;

```
const classNames = cx({
  'status-indicator--completed': props.completed,
  'status-indicator--current': props.current,
});
```

There are examples of this in the library (search for `cx`) 😄